ui/src/components/AppBar/components/UserMenu/UserMenu.tsx

```diff
@@ -1,9 +1,15 @@
 import React from 'react'
 import { Avatar, Dropdown } from 'antd'
-import UserMenuContent  from '../UserMenuContent/UserMenuContent'
+import UserMenuContent from '../UserMenuContent/UserMenuContent'
 
 export const UserMenu: React.FC = () => {
-    return <Dropdown overlay={<UserMenuContent/>}><span><Avatar size="small" alt="avatar" icon="user"/></span></Dropdown>
+    return (
+        <Dropdown overlay={<UserMenuContent />}>
+            <span>
```

Comment:
We probably should remove excessive `<span />`

Reply:
@Dergash done